app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,14 @@
+class NotificationsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    group = Group.find(params[:group_id])
+
+    if current_user == group.organizer
+      UserMailer.welcome_email(group).deliver
+      render json: { message: 'Notifications sent successfully' }
```

Comment:
Could you try to say something like `Notifications was successfully sent to 10 group members` or whatever number there is....?

Reply:
updated the message